sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/providers/BigQueryManagedIT.java

```diff
@@ -121,6 +153,8 @@ public void testStreamingStorageWriteRead() {
     // streaming write
     PCollectionRowTuple.of("input", getInput(writePipeline, true))
         .apply(Managed.write(Managed.BIGQUERY).withConfig(config));
+    assertPipelineContainsTransformIdentifier(
+        writePipeline, new BigQueryStorageWriteApiSchemaTransformProvider().identifier());
```

Comment:
So the MANAGED_UNDERLYING_TRANSFORM_URN_KEY annotation of the "Managed" transform will mention the top level BigQueryWriteSchemaTransform instead of specific implementation for the write method, right ?

Reply:
Probably, we should add a unit test for this if we don't have one already.

Reply:
> So the MANAGED_UNDERLYING_TRANSFORM_URN_KEY annotation of the "Managed" transform will mention the top level BigQueryWriteSchemaTransform instead of specific implementation for the write method, right ?

Hmmm good catch, I just switched it to mentioning the top level URN. This means we lose information on what the underlying implementation is though

Reply:
> we should add a unit test for this

Translation tests should be addressing this already